src/applications/appeals/10182/containers/ConfirmationPage.jsx

```diff
@@ -44,7 +44,7 @@ export const ConfirmationPage = () => {
         />
         <h2>Request for Board Appeal</h2>
       </div>
-      <va-alert status="success" ref={alertRef} uswds>
```

Comment:
Why are you removing the `uswds` prop? if you no longer want this prop enabled you should do `uswds="false" `since this prop will be enabled after Feb 16th.

Reply:
Hi @pmclaren19 this change is not intended. I'm working on cleaning up this branch and adding these changes back shortly